ronaldo-alves/1.II/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+    <title>Curriculum Vitae</title>
+</head>
+<body>
+   <h1>Ronaldo Cassiano Lopes Alves - Curriculum Vitae</h1>
+   <p>Casado, 36 anos, brasileiro</p>
+   <p>Avenida Dona Eliza Paciulli, 655, Parque Ibirá II</p>
+   <h2>Informações de contato</h2>
+   <p>Telefone: 35 9 9993-2812, email: ronaldorclamaster@gmail.com</p>
+   <h2>Formação e qualificação</h2>
+    <ul>
+        <li>Pós graduação em Desenvolvimento de Aplicações para Dispositivos Móveis e Cloud Computing – INATEL – Instituto Nacional de Telecomunicações <a href="http://www.inatel.br" target="#">www.inatel.br</a></li>
+        <li>Curso de especialização em C# básico <a href="http://www.softblue.com.br/site/curso/id/10/CURSO+DE+FUNDAMENTOS+DE+C_+BASICO+ON+LINE+CS10" target="#">www.softblue.com.br</a></li>
+        <li>Curso de especialização em C# avançado <a href="http://www.softblue.com.br/site/curso/id/13/CURSO+DE+C_+AVANCADO+ON+LINE+CS13" target="#">www.softblue.com.br</a></li>
+        <li>Curso de especialização em Java avançado com JavaFX <a href="http://www.softblue.com.br/site/curso/id/2/CURSO+DE+JAVA+AVANCADO+ON+LINE+JV02" target="#">www.softblue.com.br</a></li>
+        <li>Curso de especialização em Android <a href="http://www.softblue.com.br/site/curso/id/7/CURSO+DE+ANDROID:+CRIANDO+APLICATIVOS+PARA+SMARTPHONES+E+TABLETS+BASICO+AO+AVANCADO+ON+LINE+AD07" target="#">www.softblue.com.br</a></li>
+        <li>Formado em Sistema de informação pela Univás(Universidade do Vale do Sapucai)</li>
+        <li>Especialista em certificação de rede de cobre e fibra ótica conforme os padrões técnicos atuais</li>
+    </ul>
+    <h3>Experiência profissional</h3>
```

Comment:
Is it a sub-title of your "Formação e qualificação"?
Ref.: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements